icefoganalytics/elcc-data-management
# Migrate the Remaining Tables From Knex to Sequelize: FundingSubmissionLine

Relates to https://github.com/icefoganalytics/elcc-data-management/issues/29

# Context

I preformed an initial proof of concept for migrating to Sequelize, but [the PR](https://github.com/icefoganalytics/elcc-data-management/pull/19) got pretty big, so I'm migrating the models one at a time.

This PR covers migrating the following tables to a Sequelize backed models:
- [x] funding_submission_line

# Testing Instructions

1. Boot the app via `dev up`
2. Check that it boots without crashing.
3. Log in to `http://localhost:8080`
4. Check that the app doesn't crash.
